HL7-DaVinci/CDS-Library
# DMEERX-600 Home Blood Glucose F2F Encounter Questionnaire

This PR creates the F2F Encounter Questionnaire for Home Blood Glucose Monitor DME.

To test this PR, switch to the bgm-dtr branch of test-ehr repo. https://github.com/HL7-DaVinci/test-ehr/pull/34

There are several tests based on the device request orders and the diagnosis of the patient.
1. Test Non-CGM
Patient: pat014 
Device request : E0607
Verify "Treatment Plan" Section
2. Test CGM
Patient: pat014
Device request: S1031
Verify "Treatment Plan" Section
3. Test patient diabetes diagnosis has been confirmed
Patient: pat014
Verify: 
1) Verify the "Diabetes Mellitus" under Patient Diagnosis is prepopulated
2) Under "Lab validation" section, "Confirming diagnosis of Diabetes Mellitus" is not checked
4. Test patient has not existing diabetes diagnosis in EHR
Patient: pat016
Verify:
1) Verify the "Diabetes Mellitus" under Patient Diagnosis is not prepopulated
2) Under "Lab validation" section, "Confirming diagnosis of Diabetes Mellitus" is checked
